test/pylib/scylla_cluster.py

```diff
@@ -706,9 +706,9 @@ async def uninstall(self) -> None:
         self.is_dirty = True
         self.logger.info("Uninstalling cluster %s", self)
         await self.stop()
-        await asyncio.gather(*(srv.uninstall() for srv in self.stopped.values()))
-        await asyncio.gather(*(self.host_registry.release_host(Host(ip))
-                               for ip in self.leased_ips))
+        await safe_gather(*(srv.uninstall() for srv in self.stopped.values()), return_exceptions=False)
```

Comment:
why here  return_exceptions=False ?

Reply:
Good catch. I think this is some leftovers after the debugging.